packages/core-plugin-api/api-report.md

```diff
@@ -632,14 +645,39 @@ export type PluginConfig<
   routes?: Routes;
   externalRoutes?: ExternalRoutes;
   featureFlags?: PluginFeatureFlagConfig[];
+  info?: PluginConfigInfo;
   __experimentalConfigure?(options?: PluginInputOptions): {};
 };
 
+// @public
+export type PluginConfigInfo =
+  | Partial<PluginInfo>
+  | (Omit<Partial<PluginInfo>, 'packageJson'> & {
+      packageJson: LazyLoadedPackageJson;
+    })
+  | LazyLoadedPackageJson;
+
 // @public
 export type PluginFeatureFlagConfig = {
   name: string;
 };
 
+// @public
+export type PluginInfo = {
+  packageJson?: Record<string, unknown>;
+  description?: string;
+  version?: string;
+  ownerEntityRef?: string;
```

Comment:
It may be best to allow for multiple owners tbh. Not that we would want that, but it's perhaps a bit too strict to encode that into the core APIs? Thinking just `ownerEntityRefs?: string[]`.

I very much think we should aim to have entity refs as part of the ownership, so that we can link to group pages etc. We'll need to think a bit about how to achieve that without the core API handling concerns that it shouldn't. We can lean heavily on `app-defaults` to achieve that btw. It's fine for the `app-defaults` package to have pretty broad dependencies and for example set up default route bindings or something like that.

Reply:
Agreed, changed to an array (although it can be configured to a single string).

We don't need to depend on anything catalog-wise, it's just that the name ownerEntityRefs is clear it its semantics.